### ISSUE TITLE
Add release tags to project

### ISSUE THREAD

**Issue by Nukesor** [DioxusLabs/cli#113]
Hey :)

I just wanted to ask, if you could add git tags for the official releases.
This would be nice to have, so we can go ahead and add experimental packaging for a few distros.

**Comment by Karrq** [comment]
Yes, please! I trying to package this for nix and it's very hard to determine which commit corresponds to the release on crates.io!
